src/openUrlModal/view.js

```diff
@@ -29,11 +28,7 @@ function onLoad() {
       // @see https://nodejs.org/api/url.html#url_constructor_new_url_input_base
       new URL(urlInput.value); // eslint-disable-line no-new
       // If we get this far we have a valid URL
-      if (openUrlInDefaultBrowser(urlInput.value)) {
-        errorFeedback.textContent = 'URL will open in system default browser.';
-      } else {
-        errorFeedback.textContent = '';
-      }
+      errorFeedback.textContent = '';
```

Comment:
This code is for the "Open URL..." modal, which shows context help as-you-type for the URL you're entering.  Since we're removing this behavior, we no longer need this message.